php-classes/PasswordAuthenticator.class.php

```diff
@@ -5,7 +5,9 @@ class PasswordAuthenticator extends Authenticator
 {
     // configurable settings
     public static $requestContainer = '_LOGIN';
-
+    public static $enableRememberMe = true;
+    public static $rememberCookie = 'username';
+    public static $rememberDuration = 31536000; // 1 year
```

Comment:
@jtwiest let's make these clearer and more consistent:

`$rememberUsernameEnabled`
`$rememberUsernameCookieName`
`$rememberUsernameCookieDuration`